### DISCUSSION TITLE
Does it matter which ESP32-S3 GPIO pins I use?

### DISCUSSION THREAD

**Discussion by sschueller** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#548] [Q&A] [answered]
Other than the obvious ESP32-S3 pins that one need to take care with using like strapping  and PSRAM, does it matter which GPIO pins I use for which signal?

**Comment by mrcodetastic** [comment, marked as answer]
You'd best read the official ESP32-S3 technical reference documentation. But from experience you can use any general GPIO you want.
